srcCpp/RTIDDSImpl.h

```diff
@@ -6,6 +6,7 @@
  * Subject to Eclipse Public License v1.0; see LICENSE.md for details.
  */
 
+#include <stdexcept> // This header is part of the error handling library.
```

Comment:
Can we check if this could be a problem when compiling for VxWorks

Reply:
I have tested in VxWorks 7.0, x64, gcc 4.8.1,  pentium64Vx7.0gcc4.8.1. 
Build machine: bld-rhel68

Command:
```
/local/VxWorks/VxWorks-7.SR0510/wrenv.linux -p vxworks-7
setenv LD_LIBRARY_PATH /local/preship/ndds/ndds.5.3.1/unlicensed/rti_connext_dds-5.3.1/lib/pentium64Vx7.0gcc4.8.1:$LD_LIBRARY_PATH
setenv PATH /local/preship/ndds/ndds.5.3.1/unlicensed/rti_connext_dds-5.3.1/lib/pentium64Vx7.0gcc4.8.1:$PATH
git checkout feature/33-custom-type
cp -rf resource/examples_customType/key_large/srcIdl/customType srcIdl/
cp -rf resource/examples_customType/key_large/srcCpp/CustomType.cxx srcCpp/CustomType.cxx
./build.sh --clean;./build.sh --platform pentium64Vx7.0gcc4.8.1 --nddshome /local/preship/ndds/ndds.5.3.1/unlicensed/rti_connext_dds-5.3.1/ --cpp-build --customType Test

Reply:
Got it! Perfect